### DISCUSSION TITLE
Color of tztime in i3status.conf

### DISCUSSION THREAD

**Discussion by kora24** [i3/i3#6012] [Q&A] [answered]
Hi,

How can I change the color of the date and the time in the i3status.conf file? I suppose it must be somewhere here: `tztime local {
                      format = "%Y-%m-%d %H:%M:%S"
}`

Thank you.

**Comment by waterkip** [comment, marked as answer]
According to the [docs](https://i3wm.org/i3status/manpage.html#_configuration) it should be `format = "<span foreground='#B4BE82'>%Z: %R%z</span>"` however it doesn't seem to work for me.

It does work, but you need to set the following first:

```
 general {
    # This is the important bit
    markup = "pango"
    # The rest.. 
    colors = true
    # for wifi quality and disk usage things
    color_good = "#B4BE82"
    color_degraded = "#E2A478"
    color_bad  = "#85512C"
 }
```

**Reply by kora24** [reply]
Thanks a lot!
